tests/e2e/fixtures/fixture-loaders.js

```diff
@@ -455,9 +455,13 @@ const createProductAttributes = ( fixture = fixtures.Attributes() ) => {
 				)
 				.catch( () => {
 					// At this point, the attributes probably already exist. Get them and return them instead.
-					return WooCommerce.get( 'products/attributes' ).then(
-						( response ) => response.data
-					);
+					return WooCommerce.get( 'products/attributes' )
+						.then( ( response ) => response.data )
+						.then( ( data ) => {
+							return data.find(
+								( item ) => item.name === attribute.name
+							);
+						} );
```

Comment:
what's the reason behind this change? doesn't seem to be used by the new tests

Reply:
Ok, it's the fixtures loading. If we want to return all attributes wouldn't it be better to just do one call at the end?

Reply:
I didn't write these lines, but this code is executed only when there is an error during the POST request.